src/main/webapp/css/style.css

```diff
@@ -1,3 +1,300 @@
 .favourite {
   color: yellow;
 }
+
+.jumbotron {
+    text-align: center;
+    background-color:#039BE5 !important;
+}
+
+/*.card {
+    text-align: center;
+    background-color:#2C384A !important;
+    width: 300px; 
+    height: 300px;
+    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 0.43);
+   
+}*/
+
+.sdk {
+    text-align: center;
+    background-color:#2C384A !important;
+    width: 500px; 
+    height: 75px;
+    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 0.15); 
+    color: white;
+    position: relative;
+    padding: 20px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+
+.col {
+    padding: 0px 0px;
+}
+
+
+.card-body {
+   display: flex;
+   flex-direction: column;
+}
+
+.btn-group {
+    margin-top: auto;
+}
+
+
+ .card-header {
+    font-size:30px;
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+.btn-colour-1 {
+  color: #fff;
+  background-color: #2C384A;
+  border-color: #2C384A;
+  font-weight: bold;
+  letter-spacing: 0.05em;
+  border-radius: 0;
+}
+
+
+
+/* Wizard Section*/
+
+@import url('//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-theme.min.css');
+ body {
+    margin-top:30px;
+}
+.stepwizard-step p {
+    margin-top: 0px;
+    color:#666;
+}
+.stepwizard-row {
+    display: table-row;
+}
+.stepwizard {
+    display: table;
+    width: 100%;
+    position: relative;
+}
+.stepwizard-step button[disabled] {
+    opacity: 1 !important;
+    filter: alpha(opacity=100) !important;
+}
+.stepwizard .btn.disabled, .stepwizard .btn[disabled], .stepwizard fieldset[disabled] .btn {
+    opacity:1 !important;
+    color:#bbb;
+}
+.stepwizard-row:before {
+    top: 14px;
+    bottom: 0;
+    position: absolute;
+    content:" ";
+    width: 100%;
+    height: 1px;
+    background-color: #ccc;
+    z-index: 0;
+}
+.stepwizard-step {
+    display: table-cell;
+    text-align: center;
+    position: relative;
+}
+.btn-circle {
+    width: 30px;
+    height: 30px;
+    text-align: center;
+    padding: 6px 0;
+    font-size: 12px;
+    line-height: 1.428571429;
+    border-radius: 15px;
+}
+
+/*Product Release*/
```

Comment:
Add spaces before and after the comment. Please check the other comments in the file and make sure all of them have consistent styling.